source/_integrations/pushbullet.markdown

```diff
@@ -113,20 +113,21 @@ Type | Prefix | Suffix | Example
 Device | `device/` | Device nickname | `device/iphone`
 Channel | `channel/` | Channel tag | `channel/my_home`
 Email | `email/` | Contact's email address | `email/email@example.com`
+SMS | `sms/` | Contact's phone number | `sms/0612345678`
 
 If using targets, your own account's email address functions as 'send to all devices'. All targets are verified (if exists) before sending, except email.
 
 #### Example service payload
 
-```json
-{
+```yaml
```

Comment:
This now seems invalid YAML? Target is a list? Or a string?

Reply:
Fixed